tests/config/test_themes.py

```diff
@@ -16,63 +16,13 @@
     "gruvbox_dark",
     "zt_light",
     "zt_blue",
-    "gruvbox_dark24",
 }
 
 
 def test_all_themes():
     assert all_themes() == list(THEMES.keys())
 
 
-# Check built-in themes are complete for quality-control purposes
-@pytest.mark.parametrize(
-    "theme_name",
-    [
-        theme
-        if theme in expected_complete_themes
-        else pytest.param(theme, marks=pytest.mark.xfail(reason="incomplete"))
-        for theme in THEMES
-    ],
-)
-def test_builtin_theme_completeness(theme_name):
-    theme = THEMES[theme_name]
-    styles_in_theme = {style[0] for style in theme}
-
-    assert len(styles_in_theme) >= len(REQUIRED_STYLES)
-    assert all(required_style in styles_in_theme for required_style in REQUIRED_STYLES)
```

Comment:
This should be migrated in one commit, not removed in one commit and the new one added in another.